examples/asr/asr_chunked_inference/rnnt/speech_to_text_buffered_infer_rnnt.py

```diff
@@ -203,9 +213,23 @@ def main(cfg: TranscriptionConfig) -> TranscriptionConfig:
             decoding_cfg.strategy = "greedy_batch"
         decoding_cfg.preserve_alignments = True  # required to compute the middle token for transducers.
         decoding_cfg.fused_batch_size = -1  # temporarily stop fused batch during inference.
-        decoding_cfg.beam.return_best_hypothesis = True
+        decoding_cfg.beam.return_best_hypothesis = True  # return and write the best hypothsis only
```

Comment:
In the CTC version, we use a decoding in the TranscriptionConfig section as the decoding config while here we use the one extracted from the model, right?

Reply:
Yes 
we first 
` decoding_cfg = asr_model.cfg.decoding`
 then do some changes and `change_decoding_strategy`
 and then 
` cfg.decoding = decoding_cfg`